tools/node.d.ts

```diff
@@ -9,6 +9,26 @@ export interface ParseOptions {
     shebang?: boolean;
 }
 
+export interface Def {
```

Comment:
Get rid of `interface Def` altogether. There's no such data structure in Terser. This seems to be a mixture of SymbolDef and other unrelated items.